examples/split/client/src/main.rs

```diff
@@ -66,29 +69,53 @@ async fn main() -> anyhow::Result<()> {
     Ok(())
 }
 
-pub fn make_insecure_client_endpoint(bind_addr: SocketAddr) -> io::Result<Endpoint> {
+pub fn make_insecure_client_endpoint(bind_addr: SocketAddr) -> Result<Endpoint> {
     let crypto = rustls::ClientConfig::builder()
-        .with_safe_defaults()
+        .dangerous()
```

Comment:
so we went from safe_defaults to dangerous? What's that about?

Reply:
The `with_custom_certificate_verifier` function moved from the normal `rustls::ConfigBuilder` to the `rustls::client::danger::DangerousClientConfigBuilder`.
Probably because you can register a certificate verifier that simply skips verification :upside_down_face: 
This is `make_insecure_client_endpoint` after all.

So TLDR: Sensible changes in `rustls` API